libs/blocks/locui-create/input-urls/view.js

```diff
@@ -23,11 +23,11 @@ export default function InputUrls() {
     setFormData({ ...formData, [name]: value });
   }
 
-  function handleFragmentInput(event) {
-    const { value } = event.target;
-    // console.log('formInput', name, value);
-    setFragments(value);
-  }
+  // function handleFragmentInput(event) {
```

Comment:
Could you please remove the commented code? We have an additional penalty since this is delivered as is to user's browser

Reply:
Thanks Nishant. Resolved